app/assets/stylesheets/cafe_works.css.scss

```diff
@@ -0,0 +1,19 @@
+.cafe-date {
+  background: #eb7125;
+  border: 2px;
+  border-color: black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.